docs/content/docs/recipes/teams.md

```diff
@@ -118,15 +118,15 @@ export async function getContainer(id : string) : Promise<IFluidContainer> {
 
 ### Creating Fluid container in the configuration page
 
-Open the file `src/client/<your tab name>/<your tab name>Config.tsx` in your code editor. The standard Teams tab application flow goes from configuration to content page. To allow collaborations, we need to find a way to persist the container while loading into the content page. The best solution to persist the container is to append the container ID onto the `contentUrl` and `websiteUrl`, the URLs of the content page, as a query parameter. Given that the save button in the Teams configuration page is the gateway between configuration page and content page, it is a great place for us to create the container and append the container ID in the settings.
+Open the file `src/client/<your tab name>/<your tab name>Config.tsx` in your code editor. The standard Teams tab application flow goes from configuration to content page. To allow collaborations, persisting the container while loading into the content page is crucial. The best solution to persist the container is to append the container ID onto the `contentUrl` and `websiteUrl`, the URLs of the content page, as a query parameter. Given that the save button in the Teams configuration page is the gateway between configuration page and content page, it is a great place to create the container and append the container ID in the settings.
```

Comment:
```suggestion
Open the file `src/client/<your tab name>/<your tab name>Config.tsx` in your code editor. The standard Teams tab application flow goes from configuration to content page. To enable collaboration, persisting the container while loading into the content page is crucial. The best solution to persist the container is to append the container ID onto the `contentUrl` and `websiteUrl`, the URLs of the content page, as a query parameter. Since the save button in the Teams configuration page is the gateway between the configuration page and the content page, it is a great place to create the container and append the container ID in the settings.
```